lib/pure/collections/sequtils.nim

```diff
@@ -290,11 +290,11 @@ proc unzip*[S, T](s: openArray[(S, T)]): (seq[S], seq[T]) {.since: (1, 1).} =
       unzipped2 = @['a', 'b', 'c']
     assert zipped.unzip() == (unzipped1, unzipped2)
     assert zip(unzipped1, unzipped2).unzip() == (unzipped1, unzipped2)
-  result[0] = newSeqOfCap[S](s.len)
-  result[1] = newSeqOfCap[T](s.len)
-  for elem in s:
-    result[0].add(elem[0])
-    result[1].add(elem[1])
+  result[0] = newSeq[S](s.len)
+  result[1] = newSeq[T](s.len)
+  for i in 0..<s.len:
```

Comment:
```suggestion
  for i in 0 ..< s.len:
```

Reply:
There is no consensus on this one.

Reply:
I saw @narimiran make that change in one of his prior commits.

Reply:
> I saw @narimiran make that change in one of his prior commits.

Yeah I did, but @Clyybber is right and I'm not enforcing it onto others (unlike spaces around `=`).